config/crd/bases/service.binding_servicebindings.yaml

```diff
@@ -4,7 +4,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.4.1
+    controller-gen.kubebuilder.io/version: v0.6.2
```

Comment:
This is an unrelated change, right?
I think we need a check in the CI to ensure `make generate` and `make manifests` commands are run.
For now, let this change go with this PR.

Reply:
exactly, 👍

Reply:
Created PR to add the check: #20